test/protocol/integration/governance/snapshotGovernanceAdapter.spec.ts

```diff
@@ -0,0 +1,104 @@
+import "module-alias/register";
+import { Address } from "@utils/types";
+import { Account } from "@utils/test/types";
+import { ZERO, ZERO_BYTES } from "@utils/constants";
+import { SnapshotGovernanceAdapter } from "@utils/contracts";
+import { DelegateRegistry } from "@typechain/DelegateRegistry";
+import DeployHelper from "@utils/deploys";
+import {
+  addSnapshotBeforeRestoreAfterEach,
+  getAccounts,
+  getWaffleExpect,
+  getRandomAddress,
+  getSystemFixture
+} from "@utils/test/index";
+
+import { SystemFixture } from "@utils/fixtures";
+
+const expect = getWaffleExpect();
+
+describe("SnapshotGovernanceAdapter", () => {
+    let owner: Account;
+    let deployer: DeployHelper;
+    let setup: SystemFixture;
+
+    let snapshotGovernanceAdapter: SnapshotGovernanceAdapter;
+    let delegateRegistry: DelegateRegistry;
+
+    before(async () => {
+        [owner] = await getAccounts();
+
+        deployer = new DeployHelper(owner.wallet);
+        setup = getSystemFixture(owner.address);
+        await setup.initialize();
+
+    });
+
+    beforeEach(async () => {
+        delegateRegistry = await deployer.external.deployDelegateRegistry();
+        snapshotGovernanceAdapter = await deployer.adapters.deploySnapshotGovernanceAdapter(delegateRegistry.address);
+    });
+
+    addSnapshotBeforeRestoreAfterEach();
+
+    describe("Constructor", async () => {
```

Comment:
We usually put hashtags in front of function names in the description. So this would say `#constructor` and the next one would day `#getDelegateCalldata`. Adding the hashtag makes it really easy to search for tests in really big files

Reply:
Will do!